Experiment_Launcher_code/ExperimentLauncher.py

```diff
@@ -39,6 +40,7 @@ def main():
         # Instantiate software components
         video_analyzer = Video_Analyzer()
         reward_manager = RewardManager(comport_name)
+        event_logger = EventLogger()
```

Comment:
Event logger is created in the experiment manager - I think we do not need it here